src/components/shared/accountsTable/index.js

```diff
@@ -0,0 +1,135 @@
+    } = this.props;
+    const supply = networkStatus.data.supply;
+
+    return (
+      <Box main isLoading={accounts.isLoading} className="accounts-box">
+        <BoxHeader>
+          <h1>{title}</h1>
+        </BoxHeader>
+        {
+          accounts.error
+            ? (
+              <BoxContent>
+                <BoxEmptyState>
+                  <Illustration name="emptyWallet" />
+                  <h3>{emptyStateMessage || `${accounts.error}`}</h3>
+                </BoxEmptyState>
+              </BoxContent>
+            )
+            : (
+              <React.Fragment>
+                <BoxContent className={styles.content}>
+                  <Table
+                    getRowLink={account => `${routes.accounts.path}/${account.address}`}
+                    data={accounts.data}
+                    columns={[
+                      {
+                        header: t('Rank'),
+                        className: `${grid['col-xs-1']} ${grid['col-md-1']}`,
+                        id: 'rank',
+                        getValue: () => (<span className={styles.counter} />),
+                      },
+                      {
+                        header: t('Address'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-5']}`,
+                        id: 'address',
+                        getValue: account => (
+                          <AccountVisualWithAddress
+                            address={account.address}
+                            isMediumViewPort={isMediumViewPort}
+                            transactionSubject="address"
+                            showBookmarkedAddress
+                          />
+                        ),
+                      },
+                      {
+                        header: t('Balance'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-3']}`,
+                        id: 'balance',
+                        getValue: account => (
+                          <LiskAmount val={account.balance} roundTo={0} token={tokenMap.LSK.key} />
+                        ),
+                      },
+                      {
+                        header: t('Supply'),
+                        className: `${grid['col-xs-2']} ${grid['col-md-1']}`,
+                        id: 'supply',
+                        getValue: (account) => {
+                          const amount = new BigNumber(account.balance / supply * 100);
+                          return <span>{`${amount.toFormat(2)} %`}</span>;
+                        },
+                      },
+                      {
+                        header: t('Owner'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-2']}`,
+                        id: 'owner',
+                        getValue: (account) => {
+                          const delegateUsername = account.delegate ? account.delegate.username : '';
+                          const text = account.knowledge
```

Comment:
```suggestion
                          const text = account.knowledge && account.knowledge.owner
```
🐛  service 1.1.0 https://testnet-service-staging.lisk.io/api/v1/accounts/top?limit=30 always has `knowledge`, but it's empty object if not available, so now it would show
<img width="527" alt="Screenshot 2019-11-08 at 13 43 58" src="https://user-images.githubusercontent.com/1254342/68477510-f840c680-022d-11ea-9e33-04b3f055c627.png">